### ISSUE TITLE
Add help text for download files

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#1312]
![screenshot from 2016-06-30 07-23-06](https://cloud.githubusercontent.com/assets/3323703/16478727/8aa3971c-3e93-11e6-8d9e-395cc47ea233.png)

e.g. https://pypi.io/project/simplejson/#files

@ddbeck - are you able to spec what should go here?

**Comment by ddbeck** [comment]
@nlhkabu So I was going through some issues today and came across this (and an unread email informing me of it). I'm so sorry I missed it before! I didn't mean to keep you waiting.

My instinct is something pretty short: "Download the file for your platform. If you're not sure which to choose, _learn more about installing packages_." where the italicized portion is a link to the tutorial (which I have been working on lately, but we should probably talk about that elsewhere) or something else in the PyPUG.

Also, I was wondering: has any other microcopy been written yet? I'm not sure what the intended style and tone is, to be honest.

**Comment by nlhkabu** [comment]
Hi @ddbeck  - no problem... I was going to follow up on this, but have not had the time.
Your suggested text sounds great, so I'm very happy to go with this.

Regarding the other microcopy, *most* has been written, though it could probably be improved.
There is no defined voice and tone styleguide, but I've generally tried to stick to friendly, clear and simple.  Maybe we should adopt the [material design styleguide](https://material.google.com/style/writing.html), or something similar?

As always, any help you could provide in this area would be wonderful!

**Comment by ddbeck** [comment]
Yeah, the Material design style guidelines look good to me. Except that baffling "Avoid gender ambiguity" recommendation at the end (I'd prefer to strive for inclusivity over ease of translation, personally).

If there are any particular templates and the like that you'd like me to look over, feel free to point them out to me. Warehouse is a bit big and intimidating to poke around in, if I'm being honest with myself. 😳

**Comment by dstufft** [comment]
> Yeah, the Material design style guidelines look good to me. Except that baffling "Avoid gender ambiguity" recommendation at the end (I'd prefer to strive for inclusivity over ease of translation, personally).

+1 on inclusivity over ease of translation, particularly since w're not currently translating anything.

> If there are any particular templates and the like that you'd like me to look over, feel free to point them out to me. Warehouse is a bit big and intimidating to poke around in, if I'm being honest with myself. 😳

Heh, feel free to ask for help poking around too :) FWIW I *think* everything user facing (in terms of phrasing) pretty much exists entirely within ``warehouse/templates``. You can also ignore ``warehouse/templates/{legacy,rss,sitemap,robots.txt}``.

**Comment by ddbeck** [comment]
Thanks, @dstufft! I'll start looking those over soon.

**Comment by ddbeck** [comment]
OK, I spent some time looking over the templates this evening and took notes about the various bits of text that I saw. Most of it wasn't remarkable (lots of straightforward labels) or didn't need attention yet (placeholder text waiting for actual functionality to land). Everything else I noticed is below. Maybe some of these ought to manifest as actual PRs or their own issues? I'm not sure how to handle this, honestly. But here are my notes:

[`templates/index.html`](https://github.com/pypa/warehouse/blob/master/warehouse/templates/index.html)

> PyPI helps you find and install third-party software developed and shared by other members of the Python community.

Shorten to: "PyPI helps you find and install software developed and shared by the Python community."

> Top projects as downloaded by the community

I'm not sure what "top projects" means. Is this based on download counts, a curated list, or something else?

> Hot off the press: the newest projects to be released to PyPI

Shorten to: "Hot off the press: the newest project releases"

[`templates/search/results.html`](https://github.com/pypa/warehouse/blob/master/warehouse/templates/search/results.html)

The search results use single quotes to quote the user's query. Personally, I prefer double quotes to single quotes, mainly because it seems like a more common convention, but I don't have numbers to back that up.

`mystery-theme-file.html`

OK, I couldn't find where the supposed `pypi-theme` files are but regarding whatever file contains the current red banner text.

> This is a pre-production deployment of Warehouse, however changes made here WILL affect the production instance of PyPI.

Warehouse changes affect PyPI *now*, right? I think the current wording makes it sound like the behavior's coming soon. Plus the current text is a comma splice. How about: "This is a pre-production deployment of Warehouse. Changes made here affect the production instance of PyPI (pypi.python.org)."
